### ISSUE TITLE
[Feature] $SYS with a special output UI

### ISSUE THREAD

**Issue by CptHolzschnauz** [emqx/MQTTX#1211]
#### Motivation

I use both MQQT X and the MQTT explorer. One IMHO big plus on the MQTT Explorer is the vie of the $SYS topic.
While MQQTX displays these messages as others in a "stream" (Logfile and Msg display), the MQTT Explorer show these messages better to observe and in a 'static' display. Much better to debug. 

#### Detailed design

Describe the solution you'd like, a clear and concise description of what you want to happen.:

Just like it's done in the MQTT Explorer:

<img width="322" alt="Bildschirmfoto 2023-01-28 um 06 11 37" src="https://user-images.githubusercontent.com/68701579/215243377-ea37dac7-3432-4657-bf30-f953a015ac5a.png">

When there is a new message, the font size pumps for a sec and it's a good overview because it stays in place.

IMHO The $SYS topic is something special and needs to be treated special in a tool like MQTTX.

Anyway, thanks a bunch for all the work done by the contributors of MQTTX, it's a great tool and nicely shaped.

**Comment by ysfscream** [comment]
@CptHolzschnauz Hi, thank you for using MQTT X. Your suggestion is excellent and a significant feature of the MQTT Explorer for what you have shown in the screenshot. It's so good.

We will consider optimizing the display of the system topic in the future. You need a better UI view of the MQTT Broker's system metrics, right?

**Comment by CptHolzschnauz** [comment]
Cheers! You impressed my with your fast reaction, wow!

Yes, that would by nice. I'm sure you know "MQTT Explorer 0.3.5" by Thomas Nordquist where you can compare the UI.

**Comment by ysfscream** [comment]
YES. Let me do some optimisations for this one, please be patient for a later release.

**Comment by CptHolzschnauz** [comment]
Sounds good! I will..
